common/src/KokkosFFT_utils.hpp

```diff
@@ -135,15 +135,19 @@ std::size_t get_index(ContainerType& values, const ValueType& value) {
 }
 
 template <typename T, std::size_t... I>
-std::array<T, sizeof...(I)> make_sequence_array(std::index_sequence<I...>) {
+constexpr std::array<T, sizeof...(I)> make_sequence_array(
```

Comment:
I think `make_sequence_array` can now be removed

Reply:
I have removed this